src/components/Article.jsx

```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+import '../App.scss';
+
+export const Article = (props) => {
+  const { title, date, text } = props;
```

Comment:
you can do the destructuring directly instead of receiving the object

Reply:
No, it's ok